### PR TITLE
Adding OpenAPI files to documentation

### PR DESCRIPTION
**Pull request by sishida**
Signed-off-by: Sarah Ishida <sishida@us.ibm.com>

Adding OpenAPI files to the Codewind documentation.

### CODE REVIEW

File: docs/_documentations/codewind-openapi-vscode.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+2. If you have **Codewind** installed, go to the **Explorer** view group and open the **Codewind** view.
+  - Access the context menu commands from the Codewind view, go to the context menu on a project and select one of the **Generate** actions.
+3. The commands are also available from the **Command Palette** .
+  - Open the [Command Palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) and type `OpenAPI` to see the actions available.
+4. After you generate code, edit the `.openapi-generator-ignore` file to ensure that subsequent code generation does not overwrite custom code.
+
+## Features
+- Generate API clients in any of the supported [languages/frameworks](https://github.com/OpenAPITools/openapi-generator#overview).
+- Generate server stubs in any of the supported [languages/frameworks](https://github.com/OpenAPITools/openapi-generator#overview).
+- Generate HTML documentation from an OpenAPI definition file.
```
Comment (keithchong) [inline docs/_documentations/codewind-openapi-vscode.md]: @sishida , I had some inconsistencies in referring to the OpenAPI document.   I already have some PRs lined up to fix this in github.   So this line 21 is ok.

File: docs/_documentations/codewind-openapi-eclipse.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+2. If Codewind is installed, the context menu generator actions are available in the **Codewind Explorer** view.
+3. Ensure at least one OpenAPI Specification is in the project.
+4. Select an action to generate the client or server stubs or HTML documentation.
+5. Enter the requested information that is displayed in the wizard.
+6. After generation, edit the `.openapi-generator-ignore` file to ensure that subsequent code generation does not overwrite custom code.
+
+## Features
+- Generate API clients in any of the supported [languages/frameworks](https://github.com/OpenAPITools/openapi-generator#overview).
+- Generate server stubs in any of the supported [languages/frameworks](https://github.com/OpenAPITools/openapi-generator#overview).
+- Generate HTML documentation from an OpenAPI Specification file.
```
Comment (keithchong) [inline docs/_documentations/codewind-openapi-eclipse.md]: This line for eclipse is not.  It should be OpenAPI definition file.

File: docs/_documentations/codewind-openapi-eclipse.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+## Installing
+1. Download and install the latest [Eclipse IDE for Java EE Developers](https://www.eclipse.org/downloads/packages/release/) or use an existing installation. If you use Codewind OpenAPI Tools with Codewind, the earliest supported version of the Eclipse IDE is 4.11.0 (2019-03).
+2. [Optional] Install [Codewind from the Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind).
+3. Install the [Codewind OpenAPI Tools from the Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind).
+
+## Running commands
+1. Launch the context menu on any existing workspace projects or any `openapi.yaml` workspace OpenAPI Specification files from the **Package Explorer** or **Project Explorer** views.
+2. If Codewind is installed, the context menu generator actions are available in the **Codewind Explorer** view.
+3. Ensure at least one OpenAPI Specification is in the project.
```
Comment (keithchong) [inline docs/_documentations/codewind-openapi-eclipse.md]: Change Specification to definition

File: docs/_documentations/codewind-openapi-eclipse.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+The Codewind OpenAPI Tools for Eclipse includes wizards that invoke the OpenAPI Generator to create API clients, server stubs, and HTML documentation from OpenAPI Specifications. The tools are integrated and customized to work with Codewind for Eclipse, but they also work with a base Eclipse IDE for Java EE Developers installation.
+
+## Installing
+1. Download and install the latest [Eclipse IDE for Java EE Developers](https://www.eclipse.org/downloads/packages/release/) or use an existing installation. If you use Codewind OpenAPI Tools with Codewind, the earliest supported version of the Eclipse IDE is 4.11.0 (2019-03).
+2. [Optional] Install [Codewind from the Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind).
+3. Install the [Codewind OpenAPI Tools from the Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind).
+
+## Running commands
+1. Launch the context menu on any existing workspace projects or any `openapi.yaml` workspace OpenAPI Specification files from the **Package Explorer** or **Project Explorer** views.
```
Comment (keithchong) [inline docs/_documentations/codewind-openapi-eclipse.md]: Change Specification to definitions

File: docs/_documentations/codewind-openapi-vscode.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# Codewind OpenAPI Tools for VS Code
+
+The Codewind OpenAPI Tools for VS Code provides commands that invoke the OpenAPI Generator to create API clients, server stubs, and HTML documentation from OpenAPI Specifications. The tools are integrated and customized to work with Codewind for VS Code, but they can also work without the Codewind extension.
```
Comment (keithchong) [inline docs/_documentations/codewind-openapi-vscode.md]: Change Specifications to definitions

Comment (sishida) [reply docs/_documentations/codewind-openapi-vscode.md]: Addressed all "Specifications" to "definition" changes in PR https://github.com/eclipse/codewind-docs/pull/35.

**Pull request merged**